.gitignore

```diff
@@ -1,2 +1,3 @@
 # Project exclude paths
-/target/
\ No newline at end of file
+/target/
+.idea
```

Comment:
It's a good idea to add `.idea` folder to the `.gitignore` file since it's not needed for the functionality of code